language_service/src/compilation.rs

```diff
@@ -239,11 +246,61 @@ impl Compilation {
         }
     }
 
+    pub(crate) fn new_qasm(
+        package_type: PackageType,
+        target_profile: Profile,
+        package_graph_sources: PackageGraphSources,
```

Comment:
I already commented about this in a couple places now - maybe let's just drop the use of `PackageGraphSources` for QASM, and use just `sources` (i.e. the `(str,str)[]` one) instead. The "package graph" is a very Q# specific concept and you're dragging the type around only to have to fight it in QASM-land :)